### PR TITLE
Toast Documentation

### PR DESCRIPTION
**Pull request by gosteenBASIS**
Here is a draft of the Toast documentation. It will probably have to wait until the corresponding Toast demos are published, but there are placeholders for the sample programs for now.

Please add your suggestions for any improvements!

### CODE REVIEW

File: docs/components/toast.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+toast.open();
+```
+<!-- Todo: Replace with URL of published demo -->
+<ComponentDemo 
+path='http://localhost:55002/webapp/controlsamples?class=componentdemos.toastdemos.ToastPlacement' 
+javaE='https://raw.githubusercontent.com/webforj/ControlSamples/main/src/main/java/demos/webcomponents/toast/ToastPlacement.java'
+height='200px'
+/>
+
+## Actions and interactivity
```
Comment (bbrennanbasis) [inline docs/components/toast.md:65]: Capitalize the I in Interactivity.

Comment (gosteenBASIS) [reply docs/components/toast.md:65]: I believe we are using Google's style guide, which states that we should use sentence case for headings:
https://developers.google.com/style/capitalization#capitalization-in-titles-and-headings

File: docs/components/toast.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+<DocChip tooltipText="The name of the web component that will render in the DOM." label="dwc-toast" clickable={false} iconName='code'/>
+
+<!-- Todo: Is it necessary to mention the dwc-toast-group that the dwc-toast is nested under? -->
+
+<!-- Todo: Where are the Javadocs for Toast? -->
+<!-- <JavadocLink type="foundation" location="com/webforj/component/toast/Toast" top='true'/> -->
+
+The webforJ `Toast` component creates a subtle [pop-up notification](https://en.wikipedia.org/wiki/Pop-up_notification). Toast notifications disappear after a set amount of time and don't require a user response. They are typically used to provide feedback about operations or to display system messages.
+
+## Usage
```
[the remark this answers is not in the document]
Comment (gosteenBASIS) [reply docs/components/toast.md]: Thanks for the suggestion Ben; I changed this heading to "Creating a toast notification"

**Comment by gosteenBASIS** [comment]
@MatthewHawkins Can you review the updates I made to the Toast documentation?

File: docs/components/toast.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+
+<DocChip tooltipText="This component renders with a shadow DOM, an API built into the browser that facilitates encapsulation." label="Shadow" component="a" href="../glossary#shadow-dom" target="_blank" clickable={true} iconName="shadow" />
+<DocChip tooltipText="The name of the web component that will render in the DOM." label="dwc-toast" clickable={false} iconName='code'/>
+
+<!-- Todo: Is it necessary to mention the dwc-toast-group that the dwc-toast is nested under? -->
+
+<!-- Todo: Where are the Javadocs for Toast? -->
+<!-- <JavadocLink type="foundation" location="com/webforj/component/toast/Toast" top='true'/> -->
+
+A `Toast` notification is a subtle and unobtrusive [pop-up notification](https://en.wikipedia.org/wiki/Pop-up_notification) designed to provide users with real-time feedback and information. These notifications are typically used to inform users about operations such as successful actions, warnings, or errors without interrupting their workflow. `Toast` notifications typically disappear after a set amount of time and don't require a user response.
```
Comment (MatthewHawkins) [inline docs/components/toast.md:16]: Let's remove the link here

File: docs/components/toast.md [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+
+With webforJ's `Toast` component, you can easily implement these notifications to enhance user experience by providing relevant information in a familiar, non-intrusive, and seamless manner. 
+
+## Creating a toast notification
```
Comment (MatthewHawkins) [inline docs/components/toast.md:20]: In titles and in the text, please include the name of the component in backticks

File: docs/components/toast.md [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+## Styling
+
+You can style `Toast` notifications with themes just like other webforJ components, providing users with valuable context about the type of information being shown, and creating a consistent style throughout your app. You can either set the theme when you create the Toast or use the `setTheme()` method.
+
+<!-- Todo: Replace with URL of published demo -->
+<ComponentDemo 
+path='http://localhost:55002/webapp/controlsamples?class=componentdemos.toastdemos.ToastTheme' 
+javaE='https://raw.githubusercontent.com/webforj/ControlSamples/main/src/main/java/demos/webcomponents/toast/ToastTheme.java'
+height='200px'
+/>
```
Comment (MatthewHawkins) [inline docs/components/toast.md:110]: At the end, please include the <TableBuilder> component, which will build the required tables for styling information like in the other articles.

You'll need to add this entry to the _dwc_control_map.json file in components directory in docs:
"Toast":"dwc-toast",

Holler if you have a question about this!

**Review by MatthewHawkins — CHANGES_REQUESTED** [review]

File: docs/components/toast.md [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+import DocChip from '@site/src/components/DocsTools/DocChip';
+import JavadocLink from '@site/src/components/DocsTools/JavadocLink';
+import ComponentDemo from '@site/src/components/DocsTools/ComponentDemo';
+
+<DocChip tooltipText="This component renders with a shadow DOM, an API built into the browser that facilitates encapsulation." label="Shadow" component="a" href="../glossary#shadow-dom" target="_blank" clickable={true} iconName="shadow" />
+<DocChip tooltipText="The name of the web component that will render in the DOM." label="dwc-toast" clickable={false} iconName='code'/>
+
+<!-- Todo: Is it necessary to mention the dwc-toast-group that the dwc-toast is nested under? -->
+
+<!-- Todo: Where are the Javadocs for Toast? -->
```
[the remark this answers is not in the document]
Comment (gosteenBASIS) [reply docs/components/toast.md:13]: Thanks!

[the remark this answers is not in the document]
Comment (gosteenBASIS) [reply docs/components/toast.md]: Thanks!

**Review by MatthewHawkins — APPROVED** [review]

**Comment by hyyan** [comment]
@laurenic0l please update the demo URLs

**Pull request closed without merging**